core/src/toga/window.py

```diff
@@ -215,11 +241,38 @@ def size(self, size: tuple[int, int]) -> None:
     @property
     def position(self) -> tuple[int, int]:
         """Position of the window, as an ``(x, y)`` tuple."""
-        return self._impl.get_position()
+        assumed_absolute_origin = self._impl.get_primary_screen().get_origin()
```

Comment:
Why "assumed"? What's assumed about it? It's either the absolute origin or it isn't.